rs-e2e/src/main/java/gov/hhs/cdc/trustedintermediary/rse2e/AzureBlobFileFetcher.java

```diff
@@ -17,7 +17,8 @@
  */
 public class AzureBlobFileFetcher implements FileFetcher {
 
-    private static final ZoneId TIME_ZONE = ZoneOffset.UTC;
+    // Using Central Standard Time as all or most contributors are in the US
+    private static final ZoneId TIME_ZONE = ZoneOffset.of("-06:00");
```

Comment:
We want to use a time zone that is within the US for several reasons, but the most important one is that we don't want surprises when running the Automated Tests tasks. If we use UTC, that means that if someone in the PDT timezone runs `Automated Staging Test - Submit Messages` at 3:50pm and then `Automated Staging Test - Run integration tests` after 4pm, they will be running on different days, which is not expected and will make the test fail. We know that all active contributors in both TI and RS work within the US because both are federal contracts. So I think it's fair to assume all or most people working in these codebases will be in the US. The reason I picked CST instead of ET or PT is because it's in between. We want to run the task as far as possible from working hours for most of the team, so I think it makes sense not to go with either extreme

Reply:
That's a fair point, but central time is one of 2 middle US time zones.   I would suggest using Eastern or Pacific time as the default.  That way everybody only has to perform operations in one direction of the current time.

Reply:
@basiliskus that only applies if they are run manually though right? In which case we could just add documentation about that

Reply:
I don't think anyone should need to perform operations to convert timezones unless they are making changes to something related to the timezone in this code, which I don't expect will be often. The point of choosing this timezone is so people don't need to worry about it. If we choose EST, 12am EST would be 9pm PST, which is well past west coast working hours, but still possible for someone working late one day to trip over. If we pick 12am PST, the second scheduled task would run at 2am PST (2 hours after the first), which would 6am EDT (accounting for daylight savings). Again, still outside of east code working hours but still possible for someone working early in the east coast to trip over. Picking a timezone is fairly arbitrary anyway, so I'm not sure why we'd need to pick a default one in this case

Reply:
@GilmoreA6 yes, we can add it to the documentation regardless of the timezone we pick. I'm just trying to pick a timezone that would avoid as much as possible running into the issue in the first place